enterprise/rules.py

```diff
@@ -115,12 +116,49 @@ def rbac_permissions_disabled(user, obj):  # pylint: disable=unused-argument
     return not waffle.switch_is_active(ENTERPRISE_ROLE_BASED_ACCESS_CONTROL_SWITCH)
 
 
-rules.add_perm('enterprise.can_access_admin_dashboard',
-               rbac_permissions_disabled | has_implicit_access_to_dashboard | has_explicit_access_to_dashboard)
+@rules.predicate
+def open_edx_operator_has_implicit_access(user, obj):  # pylint: disable=unused-argument
+    """
+    Check that if request user has implicit access to `ENTERPRISE_OPERATOR_ROLE` feature role.
+
+    Returns:
+        boolean: whether the request user has access or not
+    """
+    request = get_request_or_stub()
+    decoded_jwt = get_decoded_jwt_from_request(request)
+    return request_user_has_implicit_access_via_jwt(decoded_jwt, ENTERPRISE_OPERATOR_ROLE, obj)
+
+
+@rules.predicate
+def open_edx_operator_has_explicit_access(user, obj):
```

Comment:
@saleem-latif Same. There is no need for this check now that Ammar's PR in edx-rbac is merged.